tests/beamlines/unit_tests/test_b16.py

```diff
@@ -0,0 +1,9 @@
+from unittest.mock import Mock
+
+from dodal.devices.b16.detector import ConstantDeadTimeController
+
```

Comment:
Could: Would be good to have a test here that `software_triggered_tiff_area_detector` gives you what you expect too

Reply:
Sure, done.